### DISCUSSION TITLE
boostrap with vite

### DISCUSSION THREAD

**Discussion by Harshitpurigoswami** [twbs/bootstrap#39762] [General]
When we use vite on you documentation and create a build it is not working please mention on your documentation how to  create build in vite bootstrap . thank  you

**Comment by darren5666** [comment]
All theme need bootstrap?

**Reply by Harshitpurigoswami** [reply]
what does it mean

**Comment by julien-deramond** [comment]
There is a build example in https://github.com/twbs/examples/tree/main/vite that corresponds to the guide, does it work for you?
If so, you'd like to have the equivalent that allows you to do a `npm run build` in [the Vite guide](https://getbootstrap.com/docs/5.3/getting-started/vite/), is that right?

**Comment by Alletkla** [comment]
Hey community :)

I join in here, cause I think I have the same problem.

I have a React Project with Bootstrap 5.3.3. I set all up like the documentation says:
https://getbootstrap.com/docs/5.3/getting-started/vite/
But I don't know why: It doesn't seem to work at least for the javascript part. CSS Is working finde.
I looked it up in an older project of mine and got it to work, but I'm curious what I (or maybe the documentation???) did wrong here.

The comment is from the Documentation. --> not working --> button is not toggable
The uncommented Part directly below is from my old project --> and works. --> button is toggable

And yeah... its not the problem that its a comment.... Its just commented for separation reasons.

```import './App.css'
// import * as bootstrap from 'bootstrap';
import 'bootstrap/dist/js/bootstrap'

function App() {

  return (
    <>
      <div className="container py-4 px-3 mx-auto">
        <h1>Hello, Bootstrap and Vite!</h1>
        <button
          type="button"
          className="btn btn-primary"
          data-bs-toggle="button"
        >
          Toggle Button
        </button>
      </div>
    </>
  )
}

export default App
```
